src/gateway.php

```diff
@@ -403,6 +403,9 @@ public function thank_you_page( $order_id ) {
 			<p>
 				<?php echo esc_html( $description ); ?>
 			</p>
+			<p>
+				<a target="_blank" href="<?php echo $GLOBALS['pay_with_ether']->base_url ?>/tutorial">Tutorial</a>
```

Comment:
I don't think `$GLOBALS['pay_with_ether']->base_url` is what you think it is. It's the URL of the plugin files on the user's installed site. Think this should just be hardcoded `https://www.paywithether.com/tutorial` ?

Also, the text needs wrapped in translation tags:

`<?php _e( 'Tutorial', 'pay_with_ether' ); ?>`